### DISCUSSION TITLE
is there a way to return DB validation errors?

### DISCUSSION THREAD

**Discussion by ccbiozhaw** [pocketbase/pocketbase#2559] [Q&A]
Hey, is there a way to return the ErrorDetail field of some requests? I am mostly looking to return the DB validation errors so i can Display them to the user?  

![image](https://github.com/pocketbase/pocketbase/assets/80820813/e885dc2f-a365-43e4-ada5-14e72bfd216f)

**Comment by ganigeorgiev** [comment]
DB errors are usually not advised to be returned directly to the end users as they could contain sensitive information.
You can inspect the error response for a public safe error message (eg. errors from fields validation). If you are using the JS SDK for example this could be done via the `err.response.data` prop:

```js
try {
    await pb.collection("example").create({...});
} catch (err) {
    console.log(err.response.data);
}
```

**Reply by ccbiozhaw** [reply]
thank you!
